Ports/gsl/package.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="gsl"
+version="2.7"
+useconfigure="true"
+files="https://mirror.freedif.org/GNU/gsl/gsl-${version}.tar.gz gsl-${version}.tar.gz"
```

Comment:
Shouldn't the linter get mad at this not having a hash?

Reply:
Thanks for pointing out, indeed a linter complained. This is my first contribution and was trying to follow what's on the Ports readme, and the example at the top didn't have an `auth_type`; the option description also says that it defaults to `md5`. Would this require an update?

I had a look now at what other sig-based GNU packages do and copied their strategy. Hopefully it will run smoothly now.